README.md

```diff
@@ -44,12 +44,28 @@ $ cd cozy-drive
 $ yarn install
 ```
 
-:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version `8` before doing a `yarn install`.
+:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version `12` before doing a `yarn install`.
 
-:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know hot to install and configure the latest available versions.
+:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know how to install and configure the latest available versions.
 
 Cozy's apps use a standard set of _npm scripts_ to run common tasks, like watch, lint, test, build…
 
+Run in dev mode:
+```sh
+$ yarn watch:drive:browser
+```
+
+Run in build mode:
```

Comment:
maybe using "build" is a bit confusing here, as we have `yarn build` for that.

Reply:
I need a little more explanation in order to fill this line. For now, I don't see the point of using yarn start:drive:browser. But maybe you can suggest me a better way to rewrite the line 🙏